<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tagMatchWidget</class>
 <widget class="QWidget" name="tagMatchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Tag</string>
   </property>
  </widget>
  <widget class="QPushButton" name="matchBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Match Tag</string>
   </property>
  </widget>
  <widget class="QListView" name="addListView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>256</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="matchListView">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>256</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="matchLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
